index.html

```diff
@@ -211,10 +212,36 @@ <h4 class="languages-heading" >Languages</h4>
       <!--Footer section-->
       <section class="footer">
         <div class="footer-div">
-          <p class="footer-paragraph">I'm always interested in hearing about <br> new projects, so <br> if you'd like to <br> chat please get <br> in touch.</p>
+          <p class="footer-paragraph">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
         </div>
       </section>
 
+      <!--Form section-->
+      <section class="form-container">
+      <form action="https://formspree.io/html" method="post" class="my-form">
+        <ul class="form-list">
+          <li class="form-list-items"><input type="text" name="userName" id="name-input" placeholder="Full name" maxlength="30" required></li>
+          <li class="form-list-items"><input type="email" name="userEmail" id="email-input" placeholder="Email address" required></li>
+          <li class="form-list-items"><textarea name="userMessage" id="message" cols="30" rows="10" maxlength="500" required>Enter text here</textarea></li>
```

Comment:
- [ ] Please from the project requirement, you should make sure that all fields have placeholder text but your `textarea` field does not have any placeholder text. So kindly add a placeholder text to the `textarea` field :pray: 
  ![image](https://user-images.githubusercontent.com/92303891/220414003-fe57ae3d-06e8-47a3-b7c6-2631c4e951ec.png)